build.sbt

```diff
@@ -30,6 +30,16 @@ lazy val migration = project
     libraryDependencies ++= Dependencies.Migration,
     publish / skip := true)
 
+lazy val tools = project
+  .in(file("tools"))
+  .enablePlugins(Publish)
+  .disablePlugins(SitePlugin, MimaPlugin)
+  .settings(
+    name := "akka-persistence-jdbc-schema-utils",
+    libraryDependencies ++= Dependencies.Migration ++ Dependencies.Libraries,
+    publish / skip := true)
+  .dependsOn(core)
```

Comment:
I first thought that you were moving the SchemaUtils to that new artifact, but now I see that it's another SchemaUtils. Adapted for the migration. That's good. 👍 

We originally decided to keep the schema utils on the core package to avoid the overhead of having yet another package. 

Anyway, we don't need to have another module. The Flyway SchemaUtils can live inside the `migration` module.

Reply:
I have provided a fix in the latest commits.